packages/schemas/src/types/domain.ts

```diff
@@ -19,3 +19,10 @@ export const domainResponseGuard = Domains.guard.pick({
 });
 
 export type DomainResponse = z.infer<typeof domainResponseGuard>;
+
+export enum DomainStatus {
+  PendingVerification = 'PendingVerification',
```

Comment:
Had a similar discussion before. Since there is no such convention recommended, as long as we follow our own standards should be fine. Add a discussion issue, we will cover this during the next weekly meet ups and come up with an aligned format convention.